ledger-core-bitcoin/idl/idl.djinni

```diff
@@ -334,6 +334,8 @@ BitcoinLikeAccount = interface +c {
     # Get addresses given a range of indices
     # Note: this will return public and change addresses
     getAddresses(from: i64, to: i64, callback: ListCallback<Address>);
+    # get all contained adresses.
+    getAllAddresses(): list<Address>;
```

Comment:
Please add `const` qualifier

Reply:
I keept the same logic as getAddresses(from, to)

Reply:
There is a lot of missing `const` qualifiers in our interface - for now we let it like that because a lot of changes are needed to add it.
However for new methods we must add that `const` qualifer.

Reply:
Adding the const qualifier will be more complicated than it seems: All nested functions must be changed to const too.

Reply:
This is a great indicator than - maybe - the name of the function is not good because a getter should - in most of the case - never mutate any member of an instance so :
- Can you list which methods/functions have to be `const` to compile the library ?
- Then if these methods/functions are not `const` by decision so rename your getter to something more appropriate

Reply:
All functions reachable by a call to getAllAddresses must be changed to const (derive, ...)
Refactoring the code could be done in another commit, but it has to touch several similar methods. 
The name must be kept as we have some similar functions like getAddresses (from, to)